sources/packages/api/src/services/education-program/education-program.service.ts

```diff
@@ -96,53 +100,71 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
     educationProgram: SaveEducationProgram,
   ): Promise<EducationProgram> {
     const program = new EducationProgram();
+
+    /** Check if education program has offering(s). This check is required to
+     *  prevent a user from updating fields that are not supposed
+     *  to be updated if the education program has 1 or more offerings.
+     */
+    const hasExistingOffering =
+      await this.educationProgramOfferingService.hasExistingOffering(
+        educationProgram.id,
+        educationProgram.locationId,
+      );
+
+    // Assign attributes for update from payload only if existing program has no offering(s).
+    if (!hasExistingOffering) {
```

Comment:
Taking the opportunity, can we please save the `creator` and `modifier` audit columns also?

Reply:
Sure. I will add them.

Reply:
Thanks for adding them.